core-libs/setup/ssr/testing/test-config-server.module.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ModuleWithProviders, NgModule, inject } from '@angular/core';
+import { TEST_CONFIG, TEST_CONFIG_COOKIE_NAME } from '@spartacus/core';
+import { REQUEST } from '../public_api';
+
+export function getCookie(cookie: string, name: string) {
+  const regExp = new RegExp('(?:^|;\\s*)' + name + '=([^;]*)', 'g');
+  const result: RegExpExecArray | null = regExp.exec(cookie);
+
+  return (result && decodeURIComponent(result[1])) || '';
+}
+
+export function parseConfigJSON(config: string) {
+  try {
+    return JSON.parse(decodeURIComponent(config));
+  } catch (_) {
+    return {};
+  }
+}
```

Comment:
`query:` Are these functions repeted to not expose them to `@spartacus/core` public API?